rest_framework/permissions.py

```diff
@@ -112,7 +112,8 @@ def has_permission(self, request, view):
         except AttributeError:
             queryset = getattr(view, 'queryset', None)
         except AssertionError:
```

Comment:
Mainly for two reasons:
1 - To honour `getattr(view, '_ignore_model_permissions', False)` behaviour. Since some views without .queryset are tolerated.
2 - Because the absence of queryset is raised few lines below with an adhoc error message.
```python
assert queryset is not None, (
    'Cannot apply DjangoModelPermissions on a view that '
)
```
I thought it would be nice to keep it that way, to be consistent with `DjangoModelPermissions`